lib/omnibus/builder.rb

```diff
@@ -644,6 +644,39 @@ def sync(source, destination, options = {})
     expose :sync
 
     #
+    # Helper method to update config_guess in the software's source
+    # directory.
+    # @param [Hash] options
+    #   Supported options are:
+    #     target [String] subdirectory under the software source to copy
+    #       config.guess.to. Default: "."
+    #     install [Array<Symbol>] parts of config.guess to copy.
+    #       Default: [:config_guess, :config_sub]
+    def update_config_guess(target: ".", install: [:config_guess, :config_sub])
```

Comment:
I think it defaults to updating both. I just didn't want to add an `:all` option :smile: